PowerUp/app/src/main/java/powerup/systers/com/vocab_match_game/VocabMatchGameActivity.java

```diff
@@ -139,10 +145,16 @@ public void run() {
                 if (latestTile < PowerUpUtils.VOCAB_TILES_IMAGES.length) {
                     startNewTile(Math.abs(r.nextInt() % 3), imageview);
                 } else if (latestTile == PowerUpUtils.VOCAB_TILES_IMAGES.length + 2){
-                    Intent intent = new Intent(VocabMatchGameActivity.this,VocabMatchEndActivity.class);
-                    intent.putExtra(PowerUpUtils.SCORE,score);
-                    finish();
-                    startActivity(intent);
+                    final Handler handler = new Handler();
+                    handler.postDelayed(new Runnable() {
+                        @Override
+                        public void run() {
+                            Intent intent = new Intent(VocabMatchGameActivity.this,VocabMatchEndActivity.class);
+                            intent.putExtra(PowerUpUtils.SCORE,score);
+                            finish();
+                            startActivity(intent);
+                        }
+                    }, 1200);
```

Comment:
Make use of PowerUpUtils.java to include any integers.

Reply:
Done!